README.md

```diff
@@ -150,35 +163,115 @@ AWS account.
 
 ### Deploying on Azure
 
-1. Find the 'Resource group' of your cluster (usually named after cluster id) inside your 'Azure subscription'
+1. Find the 'Subscription' (usually named after your installation) name and the 'Resource group' of your cluster (usually named after cluster id) inside your 'Azure subscription'
+
 2. Create 'Storage Account' on Azure ([How-to](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create)) ['Create storage account'](https://portal.azure.com/#create/Microsoft.StorageAccount)
   - 'Account kind' should be 'BlobStorage'
   - You can do it using Powershell in Azure portal. Example:
   ```
-> az storage account create `
-     --name STORAGE_ACCOUNT_NAME `
-     --resource-group RESOURCE_GROUP `
-     --sku Standard_GRS `
-     --encryption-services blob `
-     --https-only true `
-     --kind BlobStorage `
+> az storage account create \
+     --subscription SUBSCRIPTION_NAME \
+     --name STORAGE_ACCOUNT_NAME \
```

Comment:
Maybe make it explicit that this is the user's choice

```suggestion
     --name mystorage \
```

Reply:
Replaced by a variable, with should clarify it. You'll tell me what you think of it next time I push.